### PR TITLE
Implement websockets for notifying clients

### PR DESCRIPTION
**Pull request by LowBudgetMan**
Implements a websocket-based notification solution for connected clients. A remote broker can be configured by providing the broker relay properties as follows:
```yaml
broker:
  relay:
    relay-host: localhost
    relay-port: 61613
    relay-username: foo
    relay-password: bar
```
When the properties are not provided, the application defaults to the spring managed simple broker.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,4 +15,13 @@ services:
       KEYCLOAK_ADMIN_PASSWORD: 'admin'
     ports:
       - '8010:8080'
-    command: start-dev
\ No newline at end of file
+    command: start-dev
+  rabbitmq:
+    image: rabbitmq:latest
```
Comment (tsatam) [inline docker-compose.yml:20]: I'd suggest replacing this direct image reference with a reference to the Dockerfile you included in the repository:

```suggestion
    build: 
      context: ./
      dockerfile: RabbitMqDockerfile
```

File: docker-compose.yml [hunk 2]
```diff
@@ -18,4 +15,13 @@ services:
       KEYCLOAK_ADMIN_PASSWORD: 'admin'
```
Comment (tsatam) [inline docker-compose.yml:15]: I don't know if it's easily possible, but could we have this Keycloak instance automatically start up with a configured realm for local development? 
https://www.keycloak.org/server/containers#_importing_a_realm_on_startup

File: docker-compose.yml (hunk 1 above)
Comment (LowBudgetMan) [reply docker-compose.yml:20]: Oh good call, I didn't know you could do that!

File: docker-compose.yml (hunk 2 above)
Comment (LowBudgetMan) [reply docker-compose.yml:15]: Good call! I exported my local setup that just has a basic realm with postman as a client and the container will now pull that in from a volume on startup.

**Review by tsatam — APPROVED** [review]

**Pull request merged**